ggshield/cmd/status.py

```diff
@@ -29,8 +30,12 @@ def status_cmd(ctx: click.Context, **kwargs: Any) -> int:
     if not isinstance(response, HealthCheckResponse):
         raise UnexpectedError("Unexpected health check response")
 
+    # Add the API URL to the JSON output
+    json_output = response.to_dict()
+    json_output["instance"] = client.base_uri
```

Comment:
This code is only for the JSON output, I think it's better to split the `click.echo()` call instead. Something like this:

```python
if ctx_obj.use_json:
    json_output = response.to_dict()
    json_output["instance"] = client.base_uri
    click.echo(json.dumps(json_output))
else:
    # text output
    ...
```